### ISSUE TITLE
Divide by 0 error

### ISSUE THREAD

**Issue by NullVoxPopuli** [etkirsch/gmusic#26]
Not sure if this is related to any of the other ones

```
aceback (most recent call last):                                                                                                                            │
│ Songs                           File "/home/preston/Development/gmusic/gmusic/player/Player.py", line 60, in handle_message                               │
│                                                                                                                                self.notify_attachments('END')Options                                                                                                                                                   │
│   File "/home/preston/Development/gmusic/gmusic/player/Player.py", line 53, in notify_attachments                                                         │
│   2.  Vicious Delicious                                                                              attachment.handle_event(event)                       │
│   3.  So Long, Lonesome                                                                                                              File "/home/preston/Development/gmusic/gmusic/player/PlayerController.py", line 31, in handle_event                                                                               │
│   5.  The Sacred Kings                                                         self.next()                                                                │
│   6.  The Turning Years                                                                     File "/home/preston/Development/gmusic/gmusic/player/PlayerController.py", line 40, in next                                                                                                                                │
│   8.  Hello                   next_song = self.playlist.next()                                                                                            │
│   9.  On The Corner                                             File "/home/preston/Development/gmusic/gmusic/player/PlayList.py", line 16, in next       │
│   10. Too Much Butt                                                                                                                                    self.index = (self.index + 1) % len(self.contents)                                                                                                              │
│   12. Love is an ocean                      ZeroDivisionError: integer division or modulo by zero                                                         │
│   13. Reveil                                                                                                                                              │
│   14. The Son of Flynn (Remixed by Ki:Theory)                                                            
```

**Comment by NullVoxPopuli** [comment]
looks like this is the culprit:

```
    def next(self):
        '''Increments the index, but loops back if overflow'''
        self.index = (self.index + 1) % len(self.contents)
        return self.contents[self.index]

    def previous(self):
        '''Decrements the index, but loops back if overflow'''
        self.index = (self.index - 1) % len(self.contents)
        return self.contents[self.index]
```

`self.contents` is of length 0, and you can't `%` by 0

**Comment by NullVoxPopuli** [comment]
It seems to happen on the radios screen after playing the song that the radio is seeded with (I played the 'Eat it Raw' Radio, and this error came up after the song finished.


Maybe this is more an issue of what to do when the playlist is over?

**Comment by NullVoxPopuli** [comment]
not radios -- this is for songs

**Comment by NullVoxPopuli** [comment]
related: #17

**Comment by etkirsch** [comment]
should be fixed as of a964247
